source/rc.bind/bind-log/env/LOG_SCRIPT

```diff
@@ -0,0 +1,2 @@
+n4 s500000 T $LOGDIR_PATH
```

Comment:
Since `named` already provides its own timestamp, we don't need the `T` directive here.